Userland/Libraries/LibPthread/pthread_cond.cpp

```diff
@@ -39,8 +39,21 @@ int pthread_condattr_getclock(pthread_condattr_t* attr, clockid_t* clock)
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_condattr_setclock.html
 int pthread_condattr_setclock(pthread_condattr_t* attr, clockid_t clock)
 {
-    attr->clockid = clock;
-    return 0;
+    switch (clock) {
+    case CLOCK_REALTIME:
+        [[fallthrough]];
+    case CLOCK_REALTIME_COARSE:
+        [[fallthrough]];
+    case CLOCK_MONOTONIC:
+        [[fallthrough]];
+    case CLOCK_MONOTONIC_COARSE:
+        [[fallthrough]];
```

Comment:
:thinking: `fallthrough` between consecutive cases?
I think that's the first time I've seen this in serenity code.

Reply:
That's the point of fallthrough, to mark intentional case fall through. How else do you want me to write it?

Reply:
We talked about it on discord... I've removed.